model/src/payout_curve.rs

```diff
@@ -221,7 +221,7 @@ impl Announcements {
     fn new(announcements: Vec<olivia::Announcement>) -> Result<Self> {
         let announcements = announcements
             .into_iter()
-            .sorted_by(|a, b| a.id.cmp(&b.id))
+            .sorted_by(|a, b| a.id.timestamp().cmp(&b.id.timestamp()))
```

Comment:
🤔 What does this mean for already existing positions once we roll this out? Will the system be able to match them correctly?

Reply:
It is functionally equivalent, because we were ignoring all fields except for the timestamp in the previous derived implementations of `PartialOrd` and `Ord`.